src/main/resources/react/src/redux/profile/saga/workers/UserInfoPut.js

```diff
@@ -0,0 +1,20 @@
+import { call, put } from 'redux-saga/effects';
+
+import { putUserInfo as putUserInfoAsync } from '../../../../REST/Resources';
+import { fillUserInfoSync } from '../../profileAction';
+import { startFetching, stopFetching } from '../../../ui/action';
+
+export function* userInfoPut(action) {
```

Comment:
I see the same logic into UserInfoGet.js file, so why don't you create only one function for that and additionally pass request type into it?